docs/source/Advanced-Usage.md

```diff
@@ -109,13 +109,17 @@ List of some of the analyzers with optional configuration:
   * `query`: Follow according to [docs](https://www.zoomeye.org/doc#host-search), but omit `ip`, `hostname`. Eg: `"query": "city:biejing port:21"`
   * `facets`(default: Empty string): A comma-separated list of properties to get summary information on query. Eg: `"facets:app,os"`
   * `page`(default 1): The page number to paging
-  * `history`(default True):  	To query the history data.
+  * `history`(default True): To query the history data. 
+* `MWDB_Scan`:
+    * `upload_file` (default 0): Uploads the file to repository.
```

Comment:
the defaults are not the ones stated in the `analyzer_config.json` but the ones written in the new .py module. This is because `analyzer_config.json` can be modified and changed by users, while the python code cannot.

So, you should move the defaults to the python code